package.json

```diff
@@ -53,7 +53,7 @@
     "libphonenumbers": "^1.3.0",
     "mock-aws-s3": "^4.0.2",
     "moment": "^2.29.3",
-    "next": "^12.0.10",
+    "next": "^12.3.4",
```

Comment:
Please don't modify this file, a different version can affect other dependencies, so we don't want to take that risk.